src/components/batch/BatchIngest.vue

```diff
@@ -55,9 +55,6 @@
         <ul>
           <li class="error-li" v-for="(error, index) in validationResponse.processingErrors" v-bind:key="index">{{error}}</li>
         </ul>
-      </div>
```

Comment:
Was part of the story to remove this text?  Or was there another reason to remove it?

Reply:
It is not part of the Jira. The code block I removed is duplicate. it will never gonna execute, the same condition exists above this block.
please take a look here. I removed the second block.


<div v-else-if="validationResponse && !validationResponse.success " class="batch-errors">
        <p>There is an error processing your batch.</p>
        <ul>
          <li class="error-li" v-for="(error, index) in validationResponse.processingErrors" v-bind:key="index">{{error}}</li>
        </ul>
      </div>
      <div v-else-if="validationResponse && !validationResponse.success" class="batch-errors">
        <p>There is an error processing your batch.</p>
      </div>